agent/internal/containers.go

```diff
@@ -462,3 +482,36 @@ func (c *containerManager) unmakeContainerDockerLabels(ctx *actor.Context, cont
 		State:   state,
 	}, nil
 }
+
+func (c *containerManager) revalidateContainers(
```

Comment:
should we make sure the state hasn't changed, at least until `// TODO(ilia): buffer and resend pending network messages.` is addressed?

Reply:
added a check to `handleContainersReattached`